packages/runtime-vapor/src/apiCreateFor.ts

```diff
@@ -140,6 +148,13 @@ export const createFor = (
           parent!.textContent = ''
           parent!.appendChild(parentAnchor)
         }
+
+        // render fallback nodes
+        if (frag.fallback) {
+          insert((frag.nodes[0] = frag.fallback()), parent!, parentAnchor)
+          oldBlocks = []
```

Comment:
_Nitpick_: in my opinion having this `oldBlocks = []` assignment at the end of this function as an `else` case for the `!isFallback` check feels a bit more readable